docs/reference/index-modules.asciidoc

```diff
@@ -277,6 +277,17 @@ Other index settings are available in index modules:
 
     Control over the transaction log and background flush operations.
 
+<<index-modules-command-line-tools,Command-line tools>>::
+
+    Command-line tools if shard is corrupted
+
+[float]
+[[elasticsearch-shard-tool]]
+=== elasticsearch-shard tool
+
+You can use the <<index-modules-elasticsearch-shard,shard-tool>> recovery tool to remove corrupted translog or corrupted Lucene
```

Comment:
no, this part is really for the new tool - I keep `elasticsearch-translog` as it at `trasnlog.asciidoc` 

I think it has to be named just `the tool` w/o any recovery / fixing etc as after that it goes explanation

Reply:
Right, but this doesn't actually compile:

```
/Users/davidturner/src/elasticsearch-master/html_docs/index.xml:84904: element link: validity error : IDREF attribute linkend references an unknown ID "index-modules-elasticsearch-shard"
/Users/davidturner/src/elasticsearch-master/html_docs/index.xml:84894: element link: validity error : IDREF attribute linkend references an unknown ID "index-modules-command-line-tools"
```

and if it did it would render as

> shard-tool

which isn't what it should look like.

Reply:
I suppose `index-modules-shard-tool` should be better name rather just `shard-tool`